lib/templates/bootstrap/datepicker.android.js

```diff
@@ -65,6 +66,9 @@ function datepicker(locals) {
     if (locals.config.dialogMode) {
       dialogMode = locals.config.dialogMode;
     }
+    if (typeof locals.config.androidTimeIs24Hr !== "undefined") {
```

Comment:
maybe change this check to
```
if (typeof locals.config.androidTimeIs24Hr !== "undefined" && locals.config.androidTimeIs24Hr === false) {
  is24HrTimeFormat = false;
}
```
else it would be possible to pass an object/string as androidTimeIs24Hr - which might crash the app when we try to set it on android